hack/cluster-deploy.sh

```diff
@@ -31,10 +31,29 @@ function main() {
 
     # Ignore errors because some clusters might not have prometheus operator
     kubectl apply -f ${MANIFESTS_OUT_DIR} || true
-    kubectl rollout status daemonset kepler-exporter -n kepler --timeout 60s
-
-    echo "Check the logs of the kepler-exporter with:"
-    echo "kubectl -n kepler logs daemonset.apps/kepler-exporter"
+    
+    # round for 3 times and each for 60s
+    # check if the rollout status is running
+    deploy_status=1
+    for i in 1 2 3
```

Comment:
@SamYuan1990 instead of check this 3 times, why not use a larger timeout?
180s?

Reply:
will try to see any difference.

Reply:
The different will be a more clean code :)